hack/build/run-lint-checks.sh

```diff
@@ -20,11 +20,25 @@
 source hack/build/config.sh
 source hack/build/common.sh
 
+# Temporary var for pkgs that are ready to enforce linting
+LINTABLE=(pkg/controller pkg/apis pkg/client pkg/controller pkg/expectations pkg/image pkg/importer pkg/lib pkg/util)
```

Comment:
We should include `tests/...` in this as well.

Reply:
Yeah, definitely except they're not passing yet.  There's a ton of work to do in common first then the tests.  Tests are bottom of the list for now IMO, but they're not in this change because they won't pass.